api/src/timeseries.rs

```diff
@@ -127,6 +126,8 @@ pub async fn get_timeseries_data_regular(
         "P1D" => "1 day",
         _ => "1 minute", // FIXME: this should error instead of falling back to a default
     };
+
+    // TODO: this generates nulls till utc.now if end_time is not specified in the database?
```

Comment:
Yes, that's the intended behaviour. If the timeseries hasn't ended then there should be data there, so if there isn't it returns nulls for the gaps